src/components/NoteState/NoteState.js

```diff
@@ -37,15 +38,16 @@ function NoteState(props) {
   const [t] = useTranslation();
   const [isOpen, setIsOpen] = useState(openOnInitialLoad);
   const popupRef = useRef();
+  const { popupMenuRef, location } = useOverflowContainer('.normal-notes-container', 'bottom', isOpen);
 
   const isOwnedByCurrentUser = annotation.Author === core.getCurrentUser();
 
   useOnClickOutside(popupRef, () => {
-    setIsOpen(false);
+    // setIsOpen(false);
   });
 
   useOnFocusOutside(popupRef, () => {
-    setIsOpen(false);
+    // setIsOpen(false);
```

Comment:
useClickOutside and useOnFocusOutside hooks still needed?

Reply:
That was a test I did to check something in the inspect tool :sweat_smile: Nice catch!

Reply:
:eyes: